pulsar-broker/src/main/java/org/apache/pulsar/broker/service/schema/AlwaysSchemaValidator.java

```diff
@@ -0,0 +1,21 @@
+package org.apache.pulsar.broker.service.schema;
+
+import org.apache.avro.Schema;
+import org.apache.avro.SchemaValidationException;
+import org.apache.avro.SchemaValidator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * An avro schema validator that always reports as compatible.
+ */
+class AlwaysSchemaValidator implements SchemaValidator {
+    private final static Logger log = LoggerFactory.getLogger(AlwaysSchemaValidator.class);
+    static AlwaysSchemaValidator INSTANCE = new AlwaysSchemaValidator();
```

Comment:
nit: `static final ..`

Reply:
Removed the logger, wasn't used. Forgot to remove after copy paste.. Great to spot the little one 👍